pkg/okteto/client.go

```diff
@@ -165,6 +148,50 @@ func SetKubeConfig(cred *Credential, kubeConfigPath, namespace, userName, cluste
 	return clientcmd.WriteToFile(*cfg, kubeConfigPath)
 }
 
+//RemoveKubeConfig removes from a kubeconfig fle the okteto cluster credentials
+func RemoveKubeConfig(cred *Credential, kubeConfigPath, namespace, userName, clusterName string) error {
+	contextToRemoveName := getContextName(cred, namespace, clusterName)
+	cfg, err := getOrCreateKubeConfig(kubeConfigPath)
+	if err != nil {
+		return err
+	}
+
+	contextToRemove, ok := cfg.Contexts[contextToRemoveName]
+	if !ok {
```

Comment:
I think it's better not to return an error, since the objective of the call  (to remove the context) has been accomplished).